output/Form.md

```diff
@@ -40,7 +40,40 @@ title: Yerun Open Science Award form
 
 -   *Can the practice be reused in other departments or institutions?*
 
-<!--# blank field, max 400 words -->
+The idea is to record all the steps that are taken so that they are accessible
+and reusable by everyone, regardless of their level of experience with the
+technology. Thus, in our research group we are training each of the members in
+the use of tools that allow us to be more transparent and compliant with open
+science requirements in relation to our scientific production through seminars,
+such as:
+
+*The creation of projects in RStudio with a template that can be modified for
+each study we are involved.* The use of version control provided by GitHub. *The
+use of remote servers (e.g., CCC)* The use of other programming frameworks such
+as Opal to perform statistical analysis in a federated manner, so that the
+research process increases its transparency while eliminating the possibility of
+sensitive information that identifies individuals coming to light when the data
+is aggregated.
+
+In this way, each member of the group knows how to implement changes to
+manuscript versions and programming code, and where to look for track records of
+changes, so that everything could be accounted for. We get this experience from
+two projects that are either ongoing or already completed,
+[RESPOND](https://respond-project.eu/es/) and [SYNCHROS](https://synchros.eu/).
+It is not only about recording the scientific production, but also the process
+of getting to it (i.e., recording minutes of meetings).
+
+We are currently using GitHub for participanting in different research projects
+in hidden mode. GitHub is also being used for the proposal design for the (Yerun
+Open Science Award)[<https://github.com/CCOMS-UAM/yos-award>]. The aim is to be
```

Comment:
@Cristrinaranjus the links still need to be written in "plain text" instead of as "linked text". Also, strip down the `<>` symbols as they may be misinterpreted by an HTML parser (better put the link in parentheses)